modules/oe_theme_paragraphs_contact/oe_theme_paragraphs_contact.install

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/**
+ * @file
+ * Install hooks for OpenEuropa Theme Paragraphs Contact module.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Entity\Entity\EntityViewDisplay;
+
+/**
+ * Implements hook_install().
+ *
+ * Sets contacts rendering to use the full view mode.
+ */
+function oe_theme_paragraphs_contact_install() {
```

Comment:
We don't create a new view display for the paragraph, just changing the view mode of the contact entity to be rendered in full view mode instead of the default one that comes from the paragraphs module.